task2.py

```diff
@@ -0,0 +1,16 @@
+def show_student( name,GPA):
+
+    if  GPA:
+        print(GPA)
+    elif GPA==None:
+         print(0)        
+    return name + GPA
```

Comment:
You can assign a default value to the argument in the function definition `(name, GPA=0)`
So it will take the default value in the definition if you do not pass it and no need for the if condition.